admin/app/jobs/RebuildIndexJob.scala

```diff
@@ -96,7 +96,16 @@ class RebuildIndexJob(contentApiClient: ContentApiClient)(implicit executionCont
       contributors <- contentApiTagsEnumerator.enumerateTagType("contributor")
     } yield {
       val contributorsByNameOrder: Map[String, Set[Tag]] = contributors.toSet
-        .groupBy(tag => tagPages.alphaIndexKey(tag.lastName orElse tag.firstName getOrElse tag.webTitle))
+        .groupBy(tag => {
+
+          /**
+            * The alphaIndexKey function looks at the first alphanumeric character in the string passed to it.
+            * Concatenating these three strings therefore allows us to use firstName and webTitle as fallbacks
+            * if lastName is None or "".
+            * */
+          val indexString = tag.lastName.getOrElse("").trim + tag.firstName.getOrElse("").trim + tag.webTitle
```

Comment:
Is there a risk of these being duplicated between contributors? If so do we know the effect that this might have?

Reply:
good question! my understanding is that these values are just used to bucket contributors by the first letter in their names, if available. (More context on this functionality can be found in a previous PR from @ioannakok: #25176) So there shouldn't be any need for the `indexString` value to be unique, if I've understood the functionality correctly.